src/SFA.DAS.AANHub.Api.UnitTests/Controllers/CalendarEventsControllerTests/CalendarEventsControllerGetCalendarEventsTests.cs

```diff
@@ -165,7 +166,7 @@ public async Task Get_DefaultsSetsPageAndPageSizeAsExpected()
     {
         var mediatorMock = new Mock<IMediator>();
         var loggerMock = new Mock<ILogger<CalendarEventsController>>();
-        var sut = new CalendarEventsController(loggerMock.Object, mediatorMock.Object);
+        var sut = new CalendarEventsController(loggerMock.Object, mediatorMock.Object, Mock.Of<ICalendarEventsReadRepository>());
```

Comment:
Could the mock not be passed as a frozen parameter;
[Frozen] Mock calendarEventsReadRepository

Reply:
this repository is not used in the GET calls, so Mock.Of allows the wiring to continue working without any more need for setup